Clinical-Genomics/chanjo2
# Add coverage completeness levels to the coverage report

### This PR adds | fixes:
- Introduce another peace of information to the coverage report: the coverage completeness stats for each threshold level:


<img width="730" alt="image" src="https://github.com/Clinical-Genomics/chanjo2/assets/28093618/6660165e-24bb-4c0e-a0a1-840f29f0e3ab">


**How to prepare for test**:
- Deploy to cg-vm1 or install the branch locally

### How to test:
- In the browser go to the endpoint chanjo2-stage.scilifelab.se/report/demo and make sure the values of the report get populated.

### Review:

This [version](https://semver.org/) is a:
